CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ Changes are grouped as follows
 
 ## Unreleased
 
+## [2.4.3] - 2020-09-18
+- Increase raw rows lis limit to 10,000
```

Comment:
Why is this needed though?

```suggestion
- Increase raw rows list limit to 10,000
```

Reply:
The changelog entry or the change?

Reply:
the change

Reply:
speeeeeed 💨 10x rows in each request means 10x less network jumps

Reply:
Alright, but does raw allow 10k rows retrieval?